src/test/CourseNavigation.test.jsx

```
@@ -0,0 +1,112 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import CourseNavigation from "../pages/aurora-site/course-navigation";
+
+// Mock the ProgressCircle component
+jest.mock("@/components/ui/progress-circle", () => {
```

ESLint

[error] 'jest' is not defined.

(no-undef)